### ISSUE TITLE
KeyError: 'direct'

### ISSUE THREAD

**Issue by tjcim** [CravateRouge/bloodyAD#53]
Using this Dockerfile:
```
FROM python:latest
WORKDIR /usr/app/src
COPY bloodyAD /usr/app/src/
RUN pip install --no-cache-dir -r requirements.txt
CMD /bin/bash
```
I get the following error:
```
root@9a9fe77b0ade:/usr/app/src# python bloodyAD.py -u $user -d $domain -p $pass --host $dc get children --target 'Administrators'               
Traceback (most recent call last):
  File "/usr/app/src/bloodyAD.py", line 5, in <module>
    main.main()
    ~~~~~~~~~^^
  File "/usr/app/src/bloodyAD/main.py", line 168, in main
    params = {param_name: vars(args)[param_name] for param_name in param_names}
                          ~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'direct'
```

**Comment by tjcim** [comment]
Same type of issue running the example command in README:
```
root@9a9fe77b0ade:/usr/app/src# bloodyAD --host 172.16.1.15 -d bloody.local -u jane.doe -p :70016778cb0524c799ac25b439bd6a31 set password john.doe 'Password123!'
Traceback (most recent call last):
  File "/usr/local/bin/bloodyAD", line 8, in <module>
    sys.exit(main())
             ~~~~^^
  File "/usr/local/lib/python3.13/site-packages/bloodyAD/main.py", line 168, in main
    params = {param_name: vars(args)[param_name] for param_name in param_names}
                          ~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'oldpass'
root@9a9fe77b0ade:/usr/app/src# 
```

**Comment by CravateRouge** [comment]
Fixed with d77d8c0.
I normalized docstring parsing because Python 3.13 [removes whitespaces](https://docs.python.org/3/whatsnew/3.13.html#other-language-changes) from it automatically now and provoked an issue in how bloodyAD parses the functions.
You're too modern for bloodyAD with your python 3.13 xD

**Comment by tjcim** [comment]
Thanks so much! That was fast.
